mlrun/api/main.py

```diff
@@ -28,12 +28,14 @@
 import mlrun.api.utils.clients.chief
 import mlrun.api.utils.clients.log_collector
 import mlrun.errors
+import mlrun.lists
 import mlrun.utils
 import mlrun.utils.version
 from mlrun.api.api.api import api_router
 from mlrun.api.db.session import close_session, create_session
 from mlrun.api.initial_data import init_data
 from mlrun.api.middlewares import init_middlewares
+from mlrun.api.utils.clients import log_collector
```

Comment:
redundant, we already import log_collector above. 
either way try to avoid importing mlrun modules using the `from x import y` which can cause circular import.